v2/examples/spiffe-grpc-client/main.go

```diff
@@ -0,0 +1,52 @@
+package main
```

Comment:
This doesn't seem to have anything to do with gRPC? I was expecting us to set up a grpc dial using TLS based transport credentials using some config from the `tlsconfig` package or something...